scripts/probes/worker.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+# The status is provided by a local server started by
+#  ./manage.py django_q_probettp
+
+status=$(curl -s localhost:8080)
+echo "returned status ${status}"
+[[ $status == "Started" ]] || [[ $status == "Idle" ]]
```

Comment:
@zuhdil  could you please add the "Working" status here?

https://github.com/Koed00/django-q/blob/15155c7a99ea85a092b1d11da730ba0c4fa98ffc/django_q/conf.py#L197